redis/src/aio/tokio.rs

```diff
@@ -108,7 +109,12 @@ impl RedisRuntime for Tokio {
                 .use_sni(false)
                 .build()?
         } else {
-            TlsConnector::new()?
+            let mut builder = TlsConnector::builder();
+            if let Some(ca_cert) = ca_cert {
```

Comment:
In order to make this work with rustls as well (#601), I think it would be better to take `&[u8]` here and use `Certificate::from_der()`.

Reply:
updated